seatunnel-ui/src/App.tsx

```diff
@@ -15,24 +15,25 @@
  * limitations under the License.
  */
 
-import { defineComponent, computed, watch, ref } from 'vue'
+import { useSettingStore } from '@/store/setting'
+import { useThemeStore } from '@/store/theme'
+import themeList from '@/themes'
 import {
   NConfigProvider,
-  NMessageProvider,
   NDialogProvider,
+  NMessageProvider,
   darkTheme,
-  dateZhCN,
   dateEnUS,
-  zhCN,
-  enUS
+  dateZhCN,
+  enUS,
+  zhCN
 } from 'naive-ui'
-import { useThemeStore } from '@/store/theme'
-import { useSettingStore } from '@/store/setting'
+import type {
+  CustomThemeCommonVars,
+  ThemeCommonVars
+} from 'naive-ui/es/config-provider/src/interface'
+import { computed, defineComponent, watch } from 'vue'
 import { useI18n } from 'vue-i18n'
-import themeList from '@/themes'
-import type { GlobalThemeOverrides } from 'naive-ui'
-import type { Ref } from 'vue'
-import type { CustomThemeCommonVars, ThemeCommonVars } from 'naive-ui/es/config-provider/src/interface'
```

Comment:
@baicie I think these are non essential changes that need to be made, and there is no need to format them